clang/lib/DPCT/MapNames.cpp

```diff
@@ -1188,6 +1189,9 @@ void MapNames::setExplicitNamespaceMap() {
       {"ncclFloat64", std::make_shared<EnumNameRule>("oneapi::ccl::datatype::float64")},
       {"ncclDouble", std::make_shared<EnumNameRule>("oneapi::ccl::datatype::float64")},
       {"ncclBfloat16", std::make_shared<EnumNameRule>("oneapi::ccl::datatype::bfloat16")},
+      {"CUSOLVER_EIG_RANGE_ALL", std::make_shared<EnumNameRule>("'A'")},
+      {"CUSOLVER_EIG_RANGE_V", std::make_shared<EnumNameRule>("'V'")},
+      {"CUSOLVER_EIG_RANGE_I", std::make_shared<EnumNameRule>("'I'")},
```

Comment:
Should we deal with the other CUSOLVER_EIG* values:

include/cusolver_common.h:    CUSOLVER_EIG_TYPE_1 = 1,
include/cusolver_common.h:    CUSOLVER_EIG_TYPE_2 = 2,
include/cusolver_common.h:    CUSOLVER_EIG_TYPE_3 = 3
include/cusolver_common.h:    CUSOLVER_EIG_MODE_NOVECTOR = 0,
include/cusolver_common.h:    CUSOLVER_EIG_MODE_VECTOR = 1
include/cusolver_common.h:    CUSOLVER_EIG_RANGE_ALL = 1001,
include/cusolver_common.h:    CUSOLVER_EIG_RANGE_I = 1002,
include/cusolver_common.h:    CUSOLVER_EIG_RANGE_V = 1003,